api/integration_test/form_version_test.go

```diff
@@ -0,0 +1,204 @@
+package integration_test
```

Comment:
Really nice to see more inroads regarding integration tests – thanks! How do you call them?

Reply:
go test ./api/integration

Reply:
i'll add instructions to the readme

Reply:
Cool – can we get a Makefile target and/or an entry in `advanced.md`, etc. If we are having to manually run them at the moment, we probably want to require it as part of release process described in `releases.md`.

Reply:
They are being run in CI!